readme.md

```diff
@@ -0,0 +1,42 @@
+# Sample files
```

Comment:
Perhaps a more informative title like "Tutorials for Mindee Python SDK" or something similar?